### DISCUSSION TITLE
I have a question about silent alarms.

### DISCUSSION THREAD

**Discussion by jointigers** [jertel/elastalert2#1373] [Q&A] [answered]
If I have multiple rules that need to be filtered, and each rule has a different silent period every day, how should I configure it?

--rule <rule.yaml> will only run the given rule. The rule file may be a complete file path or a filename in rules_folder or its subdirectories.

--silence <unit>=<number> will silence the alerts for a given rule for a period of time. The rule must be specified using --rule. <unit> is one of days, weeks, hours, minutes or seconds. <number> is an integer. For example, --rule noisy_rule.yaml --silence hours=4 will stop noisy_rule from generating any alerts for 4 hours.

**Comment by jertel** [comment, marked as answer]
Read the docs. Search for `limit_execution`.
